docs/pages/multiselect.mdx

```diff
@@ -0,0 +1,1002 @@
+## Basic Async
+
+```jsx
+function MultiSelectExample() {
+  const [defaultOptions, setDefaultOptions] = useState([]);
```

Comment:
What is the need of defaultOptions ? Without looking at code I couldn't understand.

Reply:
![image](https://user-images.githubusercontent.com/39694575/82076856-0ea76d80-96fc-11ea-8a0b-a7e3aed5d35b.png)

Own implementation of React Select behaviour

Reply:
I will resolve this once I try the API again.